xinference/client/restful/restful_client.py

```diff
@@ -684,6 +684,37 @@ def translations(
         response_data = response.json()
         return response_data
 
+    def speech(
+        self,
+        input: str,
+        voice: str = "",
+        response_format: str = "mp3",
+        speed: float = 1.0,
+    ):
+        """Generates audio from the input text.
+
+        :param input: The text to generate audio for. The maximum length is 4096 characters.
```

Comment:
Could you use the same comment style as the other functions?

Reply:
OK